### ISSUE TITLE
Train set and Test set

### ISSUE THREAD

**Issue by primejava** [lyndonzheng/Pluralistic-Inpainting#16]
I looked at the code and found that there was no distinction between the training set and the test set in the code. Can you use this version of code for training to reproduce the current effect?

**Comment by lyndonzheng** [comment]
This is a duplicate of https://github.com/lyndonzheng/Pluralistic-Inpainting/issues/4

**Comment by primejava** [comment]
I see. Thank you for your quick response.
